src/Essentials/src/AppInfo/AppInfo.ios.tvos.watchos.macos.cs

```diff
@@ -25,6 +26,10 @@ class AppInfoImplementation : IAppInfo
 
 		public string BuildString => GetBundleValue("CFBundleVersion");
 
+		[UnsupportedOSPlatformGuard("ios13.0")]
+		private static readonly bool IsIos12TvOs13OrBefore = (OperatingSystem.IsIOS() && !OperatingSystem.IsIOSVersionAtLeast(13, 0)) ||
```

Comment:
this checks for tvOS **12**, not 13 right? and does it need `[UnsupportedOSPlatformGuard("tvos13.0")]` too?

Reply:
Yes it could have that too, just added for iOS because build was targeting iOS

Reply:
Yes but the property name is incorrect now. It should be `IsIos12TvOs12OrBefore`, though to be honest it doesn't feel like having this property is that useful, I find the original code directly in `RequestedTheme` more readable.

Reply:
The original code is causing a warning because of an analyzer bug, it is kind of a workaround for that, i will revert it when the bug is fixed and consumed by the repo

Reply:
[Bug fixed](https://github.com/dotnet/roslyn-analyzers/pull/5953) and this is reverted